qiskit/pulse/macros.py

```diff
@@ -47,7 +51,41 @@ def measure(
 
     Returns:
         A measurement schedule corresponding to the inputs provided.
+    """
+
+    # backend is V2.
+    if hasattr(backend, "target"):
+        return measure_v2(qubits, backend, target, meas_map, qubit_mem_slots, measure_name)
```

Comment:
```suggestion
    if target or hasattr(backend, "target"):
        return measure_v2(
            qubits=qubits, 
            target=target or backend.target, 
            meas_map=meas_map or getattr(backend, "meas_map", None), 
            qubit_mem_slots=qubit_mem_slots, 
            measure_name=measure_name,
        )
```
Note that backend is optional here.

Reply:
I change the code at https://github.com/Qiskit/qiskit-terra/pull/9912/commits/509392a57b805f64e7dd4030d3c6a109a64427f5